test/e2e/autoscaling/custom_metrics_stackdriver_autoscaling.go

```diff
@@ -217,6 +227,26 @@ var _ = SIGDescribe("[HPA] Horizontal pod autoscaling (scale resource: Custom Me
 			hpa:             externalHPA(f.Namespace.ObjectMeta.Name, metricTargets)}
 		tc.Run()
 	})
+
+	ginkgo.It("should scale down to 0 with Custom Metric of type Object from Stackdriver [Feature:CustomMetricsAutoscaling]", func() {
```

Comment:
do you have a pointer to where this test ran and completed successfully without being skipped?

Reply:
I do not - besides running this locally, passing `--env=KUBE_FEATURE_GATES=HPAScaleToZero=true`. Can you point me to how can I run it on some test environment, so the result would be available to share?